deployments/charts/kuma/Chart.yaml

```diff
@@ -3,7 +3,7 @@ name: kuma
 description: A Helm chart for the Kuma Control Plane
 type: application
 version: 0.4.8
-appVersion: 1.0.7
+appVersion: 1.0.8
```

Comment:
Can you revert this one? We usually do version bumps during the release process.